_summaries/wang2018dynamic_2.md

```diff
@@ -1,19 +1,41 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {wang2018Ddynamics}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: specfazhou
+score: 8/10
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+* What is the core idea? <br/>
+In this paper, the authors come up with a new neural network module called EdgeConv that can use local geometric information of point clouds, which is permutational invariant and translational invariant. By dynamically updating a graph of relationships, EdgeConv is able to perform semantic segmentation task. EdgeConv can also easily be integrated into other point cloud processing pipelines. The following picture shows the entire model architecture for classification and segmentation,<br/>
+<img src = 'model_arch.png'>
+
+
+* How is it realized (technically)? <br/>
+<img src = 'Edgeconv.png'>
+EdgeConv module is essentially one MLP together with two operations called edge feature and aggregation operation. The edge feature $$h_\Theta(x_i,x_j)$$ is a non-linear function with learnable parameters, and the aggregation operation can either be $$\Sigma$$ or $$max$$. There are five different choices of edge features and aggregation operations that have important different influences. These five different choices are shown as follows:<br/>
+<img src = 'first_choice.png'> The first choice is standard convolution <br/>
+<img src = 'second_choice.png'> The second choice only has global information, and is used in PointNet <br/>
+<img src = 'third_choice.png'> $$g$$ is Gaussian kernel and $$u$$ is distance between two points in Euclidean space <br/>
+<img src = 'forth_choice.png'> This method only provides local information <br/>
+<img src = 'fifth_choice.png'> This is the best that has both local and global information
+
+
+* How well does the paper perform? <br/>
+EdgeConv has achieved great results in classification, part segmentation, and semantic segmentation. For classification, the authors evaluate their model on the ModelNet40, which not only achieved the state-of-the-art result, but also cost much shorter time to train. For segmentation, their model archieved competitive results to other point cloud processing methods. The following three pictures show how their model performs:
+<img src = 'dyn_result1.png'>
+<img src = 'dyn_result2.png'>
+<img src = 'dyn_result3.png'>
+
+
+
+
+* What interesting variants are explored?<br/>
+One very interesting point is that you can regard PointNet, PointNet++, MoNet, and PCNN as special cases with special choices of edge features and aggregation operations. 
```

Comment:
Perhaps this belong to the earlier section where you describe different choices of edge feature and aggregation operation.  I don't think they are "variants" of the method itself, since they correspond to other existing methods in the literature.

Reply:
Yeah, I know. But it seems the paper doesn't mention other related works. But thanks for pointing it out, I will state it explicitly when I do my presentation